contracts/contracts/L2/staking/Sequencer.sol

```diff
@@ -0,0 +1,148 @@
+// SPDX-License-Identifier: MIT
+pragma solidity =0.8.24;
+
+import {Initializable} from "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
+
+import {Types} from "../../libraries/common/Types.sol";
+import {Predeploys} from "../../libraries/constants/Predeploys.sol";
+import {ISequencer} from "./ISequencer.sol";
+
+contract L2Sequencer is Initializable, ISequencer {
```

Comment:
can we make the contract name `L2Sequencer` the same as the SOL file name `Sequencer`?